main.go

```diff
@@ -213,6 +215,24 @@ func main() {
 	if err != nil {
 		logFatal(err, "error occurred with trufflehog updater 🐷")
 	}
+
+	ctx := context.Background()
+
+	var execName = "trufflehog"
+
+	ticker := time.NewTicker(900 * time.Second)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-ticker.C:
+			pid := os.Getpid()
+			err = cleantemp.CleanTempDir(ctx, execName, pid)
```

Comment:
This is now removing all temp directories except for the currently running PID. What about other instances of trufflehog? It should only clean up orphaned directories if their trufflehog PID is not running